### PR TITLE
Update deps to float back to master, add anti-affinity config

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

**Review by mattmoor — DISMISSED** [review]

**Comment by n3wscott** [comment]
@mattmoor have any idea what is up here?

**Comment by mattmoor** [comment]
No, I'd pull it down and debug

File: config/config-leader-election.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    # to actually change the configuration.
+    # leaseDuration is how long non-leaders will wait to try to acquire the
+    # lock; 15 seconds is the value used by core kubernetes controllers.
+    leaseDuration: "15s"
+    # renewDeadline is how long a leader will try to renew the lease before
+    # giving up; 10 seconds is the value used by core kubernetes controllers.
+    renewDeadline: "10s"
+    # retryPeriod is how long the leader election client waits between tries of
+    # actions; 2 seconds is the value used by core kuberntes controllers.
+    retryPeriod: "2s"
```
Comment (mattmoor) [inline config/config-leader-election.yaml:36]: Add trailing newline:
```suggestion
    retryPeriod: "2s"

```

**Comment by n3wscott** [comment]
Just missed the new rbac for leases. Added.

File: config/config-leader-election.yaml (hunk 1 above)
Comment (mattmoor) [inline config/config-leader-election.yaml:36]: Add trailing newline:
```suggestion
    retryPeriod: "2s"

```

**Review by mattmoor — APPROVED** [review]

**Pull request merged**